src/NzbDrone.Core/ImportLists/LastFm/LastFmUserSettings.cs

```diff
@@ -0,0 +1,40 @@
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.LastFm
+{
+    public class LastFmSettingsValidator : AbstractValidator<LastFmUserSettings>
+    {
+        public LastFmSettingsValidator()
+        {
+            RuleFor(c => c.UserId).NotEmpty();
+            RuleFor(c => c.Count).LessThanOrEqualTo(1000);
+        }
+    }
+
+    public class LastFmUserSettings : IImportListSettings
+    {
+        private static readonly LastFmSettingsValidator Validator = new LastFmSettingsValidator();
+
+        public LastFmUserSettings()
+        {
+            BaseUrl = "http://ws.audioscrobbler.com/2.0/?method=user.gettopartists";
+            ApiKey = "204c76646d6020eee36bbc51a2fcd810";
+        }
+
+        public string BaseUrl { get; set; }
+        public string ApiKey { get; set; }
+
+        [FieldDefinition(0, Label = "Last.fm UserID", HelpText = "User to pull artists from, blank defaults to global top artists")]
+        public string UserId { get; set; }
+
+        [FieldDefinition(1, Label = "Count", HelpText = "Number of results to pull from list (Max 1000)", Type = FieldType.Number)]
```

Comment:
It looks like this defaults to 0. Do we want to change that to a better default value? Tag import list does the same.

Reply:
Crap.. yea meant to do that after testing